src/test_xor128.rs

```diff
@@ -682,56 +782,46 @@ fn test_compound_levelcache_octrees_trees_from_store_configs() {
     test_compound_levelcache_tree_from_store_configs::<U8, U7>(4096);
 }
 
-#[test]
```

Comment:
What happened to this test?

Reply:
Thanks for paying attention on this. It seems to me that this test is actually duplicated by `test_compound_quad_trees_from_slices`. We discussed with @cryptonemo, that it makes sense to use generic test like this:

```
fn test_tree() {
   fn run_test() {
      ...
   }
   run_test::<_, _, U8>
   run_test::<_, _, U4>
}
``` 

instead of two tests for quad and oct trees with actually similar logic:
```
fn test_oct_tree() {}
fn test_quad_tree() {}
```

In this PR I didn't introduced this change - only manually analysed what tests are redundant. This will be introduced in future PRs.